test/persistence-hooks.suite.js

```diff
@@ -1668,7 +1668,6 @@ module.exports = function(dataSource, should, connectorCapabilities) {
             } else {
               triggered.should.eql([
                 'access',
-                'loaded',
```

Comment:
How comes `loaded` is no longer triggered? Is that why we are implementing this fix? In that case I think you can get rid of `if (dataSource.connector.updateOrCreate)` because both branches report the same hooks. Correct?